orcid-web/src/main/resources/freemarker/common/.fuse_hidden00001af10000fdac

```diff
@@ -0,0 +1,199 @@
+<#--
```

Comment:
hidden file?

Reply:
It looks it is a hidden file, I just removed it